packages/react-data-grid/src/RowGroup.tsx

```diff
@@ -1,15 +1,15 @@
 /* eslint-disable @typescript-eslint/no-explicit-any */
 import React, { forwardRef } from 'react';
 import { EventTypes } from './common/enums';
-import { CellMetaData, IRowRendererProps, CalculatedColumn } from './common/types';
+import { IRowRendererProps, CalculatedColumn, RowExpandToggleEvent, Omit, CellRendererProps } from './common/types';
 import EventBus from './EventBus';
 
 interface Props<R> {
+  ref?: React.Ref<HTMLDivElement>;
```

Comment:
`ref` Refs shouldn't be on the `Props` type. otherwise you end up with `props.ref` being possible type-wise, when it's not in practise.

Reply:
I made a small change, let me know if there is a better way

Reply:
Yup looks good.